client/src/views/loading.js

```diff
@@ -1,11 +1,10 @@
 import Snabbdom from 'snabbdom-pragma'
 import layout from './layout'
 
+
 export default ({ t, ...S }) => layout(<div>
-  <div className="loading-delay">
```

Comment:
The `loading-delay` class was being used to make the loading indicator visible only when loading takes longer than 0.3s, so that it doesn't show up when it loads very quickly. Do we want to preserve this behavior?

It can be done by adding these CSS properties to `.spinner`:

```css
visibility: hidden;
animation: 0.3s showDelayed;
animation-fill-mode: forwards;
```